### ISSUE TITLE
text fails when texted number is a business

### ISSUE THREAD

**Issue by lcgrz** [Field-Team-6/gv-byop#1]
![image (1)](https://github.com/FieldTeam6/gv-byop/assets/9267572/fd515b22-bb0c-4465-9012-032d9ce502de)

**Comment by sttyerase** [comment]
Do you have samples of business phone numbers in the test database?

**Comment by lcgrz** [comment]
no, i already have a fix for this running locally though.  i was able to simulate with a phone bank containing my own phone number.  since i have my own number in my contacts list, it would show my name instead of the phone number up in the header section.
